src/libdredd/include/libdredd/mutation_replace_unary_operator.h

```diff
@@ -54,8 +56,14 @@ class MutationReplaceUnaryOperator : public Mutation {
   [[nodiscard]] bool IsValidReplacementOperator(
       clang::UnaryOperatorKind op) const;
 
+  std::string getExpr(clang::ASTContext& ast_context) const;
+
```

Comment:
Should be called `GetExpr` to conform to Google style.

Reply:
Done.